Buddies/Activities/ActivityTableVC.swift

```diff
@@ -7,84 +7,125 @@
 //
 
 import UIKit
+import Firebase
 
 class ActivityTableVC: UITableViewController {
-
+    
+    var stopListeningToUser: Canceler?
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        self.tableView.rowHeight = 100
         // Uncomment the following line to preserve selection between presentations
         // self.clearsSelectionOnViewWillAppear = false
-
+        
         // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
         // self.navigationItem.rightBarButtonItem = self.editButtonItem
     }
-
+    
     // MARK: - Table view data source
-
+    
     override func numberOfSections(in tableView: UITableView) -> Int {
         // #warning Incomplete implementation, return the number of sections
-        return 0
+        return 1
     }
-
+    
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         // #warning Incomplete implementation, return the number of rows
-        return 0
+        return 1
     }
-
-    /*
+    
+    //USES TEST ACTIVITY RN
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "reuseIdentifier", for: indexPath)
-
+        
+        let cell = tableView.dequeueReusableCell(withIdentifier: "activityCell", for: indexPath) as! ActivityCell
+        
+        self.stopListeningToUser = DataAccessor.instance.useActivity(id: "EgGiWaHiEKWYnaGW6cR3"){ activity in
+            cell.titleLabel.text = activity.title
+            cell.descriptionLabel.text = activity.description
+            cell.locationLabel.text = String(activity.location.latitude) + ", " + String(activity.location.longitude)
+            cell.dateLabel.text = activity.endTime.dateValue().description
+            
+            for memberID in activity.members{
+                self.stopListeningToUser = DataAccessor.instance.useUser(id: memberID){ user in
```

Comment:
this will overwrite old cancel callbacks.
we will only have the last one.

`stopListeningToUser` should probably be a list. My recomendation would be to map members to cancel callbacks:

```swift
this.stopListeningToUsers = activity.members.map { 
    DataAccessor.instance.useUser(id: $0) { user in 
        /**/
    }
}
```